src/python/pants/init/plugin_resolver.py

```diff
@@ -22,6 +21,7 @@
 from pants.util.contextutil import temporary_dir
 from pants.util.dirutil import safe_mkdir, safe_open
 from pants.util.memo import memoized_property
+from pants.util.strutil import ensure_text
 from pants.version import PANTS_SEMVER
 
```

Comment:
This was breaking with `newstr does not support decode`, which kind of makes sense.